index.html

```diff
@@ -37,6 +34,113 @@ <h3>I’am  web a developer.</h3>
             <li><img src="images/medium.png" alt="social-icon" /></li>
       </ul>   
     </section>
+       <section class="my-works1">
+
+
+        <h1>My Recent works</h1>
+
+     
+        <h3>Multi-Post Stories</h3>
+        <a class="img1"><img src="images/placeholder.png" alt="hamburger Icon" /></a>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </h2>
+        <div class="lenguajes">
+            <div class="box box1">css</div>
+            <div class="box box2">html</div>
+            <div class="box box3">bootstrap</div>
+            <div class="box box4">ruby</div>
+          
+          </div>
```

Comment:
As semantic best practice to list items, I suggest you use the unordered list tag (ul ) and li for individual items 👍